.github/workflows/CI.yml

```diff
@@ -43,30 +49,73 @@ jobs:
         curl -LsSf https://github.com/taiki-e/cargo-hack/releases/latest/download/cargo-hack-x86_64-unknown-linux-gnu.tar.gz | tar xzf - -C ~/.cargo/bin
     - name: cargo hack check
       working-directory: ${{ matrix.subcrate }}
-      run: cargo hack check --each-feature --no-dev-deps --all
+      run: cargo hack check --each-feature --no-dev-deps --workspace
 
   test-versions:
     # Test against the stable, beta, and nightly Rust toolchains on ubuntu-latest.
     needs: check
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [stable, beta, nightly, 1.40.0]
```

Comment:
tower-http doesn't declare an MSRV and it never compiled on 1.40 in the first place. Going to 1.51 for const generics.

Reply:
But we still don't specify MSRV anywhere. Think I wanna keep it like that for now.